web_responsive_pwa/controllers/main.py

```diff
@@ -0,0 +1,25 @@
+from odoo.http import request, Controller, route
+
+
+class PWA(Controller):
+
+    @route('/service-worker.js', type='http', auth="public")
```

Comment:
According to a comment down below, possibly this controller should be renamed to something like this:

```suggestion
    @route('/web_responsive_pwa/cache.json', type='json', auth="public")
```

However, if it were to stay as it is, it should be renamed too:

```suggestion
    @route('/web_responsive_pwa/service-worker.js', type='http', auth="public")
```